src/transport/raw/MessageHeader.h

```diff
@@ -71,6 +71,8 @@ static_assert(detail::kMaxApplicationPayloadAndMICSizeBytes > kMaxTagLen, "Need
 // tag we will not have source/destination node IDs, but above we are including
 // those in the header sizes.
 static constexpr size_t kMaxAppMessageLen = detail::kMaxApplicationPayloadAndMICSizeBytes - kMaxTagLen;
+// large payload limit
+static constexpr size_t kLargePayloadMaxMessageSizeBytes = 128000;
```

Comment:
We should make the Max TCP payload size a CHIPConfig or a TCPConfig and use it here. I have a bunch of TCP configs that are currently in CHIPConfig.h. I should move those to a TCPConfig.h and create a separate PR..

Reply:
Sure, we can have the move as part of the next PR when we move all the TCP related changes in one config